generate-github-app-token-from-vault-secrets/README.md

```diff
@@ -18,6 +18,8 @@ This composite GHA can be used in any repository.
 | vault-auth-secret-id                | The Secret Id for (Vault) App Role authentication                       |
 | vault-url                           | The URL for the Vault endpoint                                          |
 | skip-token-revoke                   | If truthy, the token will not be revoked when the current job is complete (optional) |
+| owner                               | The owner of the GitHub App installation (defaults to current repository owner, optional). To leave this value empty in the token generation, set it to `!null`. |
```

Comment:
why not just empty string "" instead? I think it's more intuitive like that.

Reply:
I'd also argue that there's the need to have the case of generating a token for all repositories - normally it's either the current one, or a different (specific) one, wdyt?

Reply:
basically, wouldn't it be enough to have a thin wrapper for the https://github.com/actions/create-github-app-token and just to pass `owner` and `repositories` to it as is?

Reply:
Thank you, I didn't think of the Renovate use case, it makes total sense.